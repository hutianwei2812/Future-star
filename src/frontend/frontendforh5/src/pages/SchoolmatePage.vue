<template>
  <div class="schoolmatepage">
    <title-bar :is-on-load="isOnLoad"></title-bar>
    <nav-bar :special-color="specialColor"></nav-bar>
    <br />
    <SchoolmateTable />
    <br />
    <br />
    <br />
    <br />
    <br />
    <br />
  </div>
</template>

<script>
import TitleBar from '@/components/TitleBar.vue'
import NavBar from '@/components/NavBar.vue'
import SchoolmateTable from '@/components/SchoolmateTable.vue'
export default {
  name: 'SchoolmatePage',
  data() {
    return {
      isOnLoad: true,
      specialColor: 4
    }
  },
  components: {
    SchoolmateTable,
    TitleBar,
    NavBar
  },
  props: {}
}
</script>

<style scoped type="type/css">
</style>