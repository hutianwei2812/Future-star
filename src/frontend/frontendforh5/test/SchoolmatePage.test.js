import { mount } from '@vue/test-utils'
import SchoolmatePage from '@/pages/SchoolmatePage'
import Vue from 'vue'
import ElementUI from 'element-ui'
Vue.use(ElementUI)

test('renders correctly', () => {
  const wrapper = mount(SchoolmatePage)
  expect(wrapper.element).toMatchSnapshot()
})