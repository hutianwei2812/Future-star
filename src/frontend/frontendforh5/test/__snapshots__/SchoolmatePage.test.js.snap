// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="schoolmatepage"
>
  <header
    class="el-header"
    style="height: 60px;"
  >
    <span
      class="title-span"
    >
      未来之星管理系统
    </span>
     
    <span
      class="user-span"
    >
      
    </span>
     
    <a
      class="el-link el-link--primary is-underline"
    >
      <!---->
      <span
        class="el-link--inner"
      >
        注销
      </span>
      <!---->
    </a>
  </header>
   
  <div
    class="nav"
  >
    <section
      class="el-container is-vertical"
    >
      <header
        class="el-header"
        style="height: 60px;"
      >
        <div
          class="newDiv"
        >
          <ul
            class="sub_nav"
          >
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                报名管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                课程管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                活动管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="blueColor"
              >
                校友管理
              </li>
            </div>
             
            <div>
              <li
                class="sub_nav_li"
                id="normalColor"
              >
                管理员权限设置
              </li>
            </div>
          </ul>
        </div>
      </header>
    </section>
  </div>
   
  <br />
   
  <div>
    <div
      class="matetable"
    >
      <br />
       
      <div
        class="el-table widthSet el-table--fit el-table--striped el-table--enable-row-hover"
      >
        <div
          class="hidden-columns"
        >
          <div>
            <div />
              
            <div />
             
            <div />
          </div>
        </div>
        <div
          class="el-table__header-wrapper"
        >
          <table
            border="0"
            cellpadding="0"
            cellspacing="0"
            class="el-table__header"
          >
            <colgroup />
            <thead
              class=""
            >
              <tr
                class=""
              />
            </thead>
          </table>
        </div>
        <div
          class="el-table__body-wrapper is-scrolling-none"
        >
          <table
            border="0"
            cellpadding="0"
            cellspacing="0"
            class="el-table__body"
          >
            <colgroup />
            <tbody>
              <!---->
            </tbody>
          </table>
          <div
            class="el-table__empty-block"
          >
            <span
              class="el-table__empty-text"
            >
              暂无数据
            </span>
          </div>
          <!---->
        </div>
        <!---->
        <!---->
        <!---->
        <!---->
        <div
          class="el-table__column-resize-proxy"
          style="display: none;"
        />
      </div>
    </div>
  </div>
   
  <br />
  <br />
  <br />
  <br />
  <br />
  <br />
</div>
`;
